maths/pi_monte_carlo_estimation.py

```diff
@@ -0,0 +1,64 @@
+import random
+
+
+class Point:
+    def __init__(self, x: float, y: float) -> None:
+        self.x = x
+        self.y = y
+
+    def is_in_unit_circle(self) -> bool:
+        """
+        True, if the point lies in the unit circle
+        False, otherwise
+        """
+        return (self.x ** 2 + self.y ** 2) <= 1
+
+    @classmethod
+    def random_unit_square(cls):
+        """
+        Generates a point randomly drawn from the unit square [0, 1) x [0, 1).
+        """
+        x = random.random()
+        y = random.random()
+
+        return cls(x, y)
```

Comment:
```suggestion
        return cls(x = random.random(), y = random.random())
```
and lose lines 21, 22, 23.  Then I think we can land this one.